src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -159,6 +159,15 @@ private void handleDirectoryChange(Directory directory) {
         }
     }
 
+    private void handleCostChange(Directory directory) {
```

Comment:
I think the travelPlanPanel.update() can be called here instead? So, no need to pass into TravelPlanObjectListPanel